Rakefile

```diff
@@ -115,7 +115,11 @@ task :build => [
 task :install => :build do |t|
   mkdir_p "#{DATADIR}/foreman-installer"
   cp_r Dir.glob('{checks,config,hooks,VERSION,README.md,LICENSE}'), "#{DATADIR}/foreman-installer"
+
   cp "#{BUILDDIR}/foreman-hiera.conf", "#{DATADIR}/foreman-installer/config/foreman-hiera.conf"
+  mkdir_p "#{SYSCONFDIR}/foreman-installer"
+  ln_s "#{DATADIR}/foreman-installer/config/foreman.hiera/custom.yaml", "#{SYSCONFDIR}/foreman-installer/custom-hiera.yaml"
```

Comment:
Well, it was *meant* exactly as its written now, it seemed more straightforward to leave the file where it was.

Updated to use your suggestion.